config/environments/production.rb

```diff
@@ -97,12 +97,21 @@
   #
   # The `database_resolver` class is used by the middleware to determine which
   # database is appropriate to use based on the time delay.
-  #
+  config.action_mailer.default_url_options = { host: 'erp-1.herokuapp.com' }
+
   # The `database_resolver_context` class is used by the middleware to set
   # timestamps for the last write to the primary. The resolver uses the context
   # class timestamps to determine how long to wait before reading from the
   # replica.
-  #
+  ActionMailer::Base.smtp_settings = {
+    :port           => ENV['MAILGUN_SMTP_PORT'],
+    :address        => ENV['MAILGUN_SMTP_SERVER'],
+    :user_name      => ENV['MAILGUN_SMTP_LOGIN'],
+    :password       => ENV['MAILGUN_SMTP_PASSWORD'],
```

Comment:
I like to use rails credentials instead of environment variables -> its easier to move applications from one server to another. But this is just suggestion.